typewise_alert.test.py

```diff
@@ -4,8 +4,19 @@
 
 class TypewiseTest(unittest.TestCase):
   def test_infers_breach_as_per_limits(self):
-    self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
-
-
+      #Check Breaches
+      self.assertTrue(typewise_alert.infer_breach(10, 20, 60) == 'TOO_LOW')
+      self.assertTrue(typewise_alert.infer_breach(70, 20, 60) == 'TOO_HIGH')
+      self.assertTrue(typewise_alert.infer_breach(50, 20, 60) == 'NORMAL')
+   
+   #Check Alerts
+  def test_check_and_alert(self): 
+      self.assertTrue(typewise_alert.check_and_alert('TO_EMAIL', 'PASSIVE_COOLING', 60) == 'Temperature is too high')
+      self.assertTrue(typewise_alert.check_and_alert('TO_CONTROLLER', 'PASSIVE_COOLING', 60) == '65261, TOO_HIGH')
+      self.assertTrue(typewise_alert.check_and_alert('TO_EMAIL', 'PASSIVE_COOLING', -5) == 'Temperature is too low')
```

Comment:
These asserts say that check_and_alert converts 3 parameters to one or two return values.
If you had to restate its functionality in terms of data-conversion, how would you put it?